### PR TITLE
Allow add course key configuration

### PR DESCRIPTION
**Pull request by luismorenolopera**
In other course settings it is optional to add the course key course.

Configuration with course key
```
"SAFE_EXAM_BROWSER":{
    "course-v1:seb-openedx+course_2+2019":{
      "PERMISSION_COMPONENTS":[
        "AlwaysAllowStaff",
        "CheckSEBHashBrowserExamKey",
        "CheckSEBHashConfigKey"
      ],
      "BROWSER_KEYS":[
        "cd8827e4555e4eef82........5088a4bd5c9887f32e590"
      ],
      "CONFIG_KEYS":[
        "9887f32e590cd8827e........5088a4bd5c4555e4eef82"
      ],
      "USER_BANNING_ENABLED":true
    }
}
```

Configuration without course key:
```
"SAFE_EXAM_BROWSER":{
      "PERMISSION_COMPONENTS":[
        "AlwaysAllowStaff",
        "CheckSEBHashBrowserExamKey",
        "CheckSEBHashConfigKey"
      ],
      "BROWSER_KEYS":[
        "cd8827e4555e4eef82........5088a4bd5c9887f32e590"
      ],
      "CONFIG_KEYS":[
        "9887f32e590cd8827e........5088a4bd5c4555e4eef82"
      ],
      "USER_BANNING_ENABLED":true
    
}
```

both must work

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -195,6 +195,9 @@ This is done on a per-course basis and can be modified in 3 different locations
 
     .. image:: images/other_course_settings_data.png
 
+    You can skip the course key
```
Comment (felipemontoya) [inline docs/usage.rst]: This should be the preferred way. Can you put it on top of the other.

Then you can say, "You could optionally use the course key as well" ...

File: seb_openedx/seb_keys_sources.py [hunk 2]
```diff
@@ -23,7 +23,11 @@ def from_other_course_settings(course_key):
     course_module = get_course_module(course_key, depth=0)
     if hasattr(course_module, 'other_course_settings'):
         other_settings = course_module.other_course_settings
-        return other_settings.get('SAFE_EXAM_BROWSER', None)
+        other_settings = other_settings.get('SAFE_EXAM_BROWSER', {})
+        if course_key.html_id() in other_settings:
```
Comment (felipemontoya) [inline seb_openedx/seb_keys_sources.py]: In the rest of the codebase we use `six.text_type(course_key)` to convert in a string type. Just for consistency, lets use that.

**Review by felipemontoya — CHANGES_REQUESTED** [review]
I tested this and it works like a charm.
Two minor changes and we are ready. (you can ammend/squash the changes)

File: docs/usage.rst (hunk 1 above)
Comment (luismorenolopera) [reply docs/usage.rst]: done

**Review by felipemontoya — APPROVED** [review]
From my perspective this is ready to merge. But we are lowering the code coverage here. We should add one test for this.

**Pull request merged**